src/Common/src/Microsoft/Internal/Requires.cs

```diff
@@ -4,15 +4,14 @@
 
 using System;
 using System.Collections.Generic;
-using System.Composition;
 using System.Diagnostics;
 using System.Diagnostics.Contracts;
 using System.Globalization;
 using System.Reflection;
 
 namespace Microsoft.Internal
 {
-    internal static class Requires
+    internal static partial class Requires
     {
         [DebuggerStepThrough]
         [ContractArgumentValidator]
```

Comment:
You can remove all `[ContractArgumentValidator]` attributes as we don't use contracts. Keep the code though, unless it's dead.